src/app/pages/loading-states/data.ts

```diff
@@ -0,0 +1,148 @@
+export type LoadingData = {
+    name: string;
+    data?: {
+        heroValue: number;
+        loadValue: number;
+        battery: number;
+        channels: Array<{
+            icon: string;
+            label: string;
+            value: number;
+            unit: string;
+        }>;
+    };
+};
+
+export const dataList: LoadingData[] = [
+    {
+        name: 'placeholder1',
+        data: {
+            heroValue: 0,
```

Comment:
maybe rename heroValue to scoreValue, since all of these are hero values

Reply:
Renamed scoreValue to heroValue